mantidproject/mantid
# QSettings saves to .ini across workbench

**Description of work.**
Added a line to mainwindow.py that should make QSettings save to .ini format by default which solves the issue of StartLiveData saving to a .conf and potentially others.

**To test:**
- Run StartLiveData and wait for it to run.
- Once ran check ```~/.config/mantidproject/``` make sure there is no mantidworkbench.conf and that the following lines are in the .ini file
```
[LiveDataAlgorithms]
LastDirectory=

[LiveDataPostProcessingAlgorithms]
LastDirectory=
```



Fixes #23711 . 

Workbench apparently doesn't need release notes

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
